docs/infrastructure.md

```diff
@@ -38,6 +41,10 @@ Deploys of `master` happen automatically after Travis CI tests are successful.
     If not provided, requests to the GitHub API are unauthenticated.
     The token we use in production is associated with the `nextstrain-bot` GitHub user.
 
+  - `CANARY_ORIGIN` is the origin of a canary deployment to redirect to for users who are opted-in to the `canary` flag (the `!flags/canary` Cognito group).
+    Redirection does not happen if this variable is not defined or if the request's origin already matches this variable's value.
+    In production we set this to `https://next.nextstrain.org`.
+
```

Comment:
Should `NODE_ENV` also be documented here?

Reply:
This is an Express/Node convention that we didn't set, but we do use it ourselves, so documented!